frontend/components.d.ts

```diff
@@ -7,99 +7,100 @@ export {}
-    SettingsDialog: (typeof import('./src/components/Settings/SettingsDialog.vue'))['default']
-    SettingsTab: (typeof import('./src/components/Settings/SettingsTab.vue'))['default']
-    Tabs: (typeof import('./src/components/Tabs.vue'))['default']
-    TaskDetail: (typeof import('./src/components/TaskDetail.vue'))['default']
-    TaskList: (typeof import('./src/components/TaskList.vue'))['default']
-    TaskPriorityIcon: (typeof import('./src/components/icons/TaskPriorityIcon.vue'))['default']
-    TaskStatusIcon: (typeof import('./src/components/icons/TaskStatusIcon.vue'))['default']
-    TeamMembers: (typeof import('./src/components/TeamMembers.vue'))['default']
-    TextEditor: (typeof import('./src/components/TextEditor.vue'))['default']
-    UnsplashImageBrowser: (typeof import('./src/components/UnsplashImageBrowser.vue'))['default']
-    UserAvatar: (typeof import('./src/components/UserAvatar.vue'))['default']
-    UserDropdown: (typeof import('./src/components/UserDropdown.vue'))['default']
-    UserImage: (typeof import('./src/components/UserImage.vue'))['default']
-    UserInfo: (typeof import('./src/components/UserInfo.vue'))['default']
-    UserProfileLink: (typeof import('./src/components/UserProfileLink.vue'))['default']
+    Activity: typeof import('./src/components/Activity.vue')['default']
+    AddMemberDialog: typeof import('./src/components/AddMemberDialog.vue')['default']
+    AddTeamDialog: typeof import('./src/components/AddTeamDialog.vue')['default']
+    AppSidebar: typeof import('./src/components/AppSidebar.vue')['default']
+    ArchivedTeams: typeof import('./src/components/Settings/ArchivedTeams.vue')['default']
+    AssignUser: typeof import('./src/components/AssignUser.vue')['default']
+    Bookmarks: typeof import('./src/components/Settings/Bookmarks.vue')['default']
+    ChevronTriangle: typeof import('./src/components/icons/ChevronTriangle.vue')['default']
+    ColorPicker: typeof import('./src/components/ColorPicker.vue')['default']
+    CommandPalette: typeof import('./src/components/CommandPalette/CommandPalette.vue')['default']
+    Comment: typeof import('./src/components/Comment.vue')['default']
+    CommentEditor: typeof import('./src/components/CommentEditor.vue')['default']
+    CommentsArea: typeof import('./src/components/CommentsArea.vue')['default']
+    CommentsList: typeof import('./src/components/CommentsList.vue')['default']
+    Component: typeof import('./src/components/TextEditorTaskExtension/Component.vue')['default']
+    CoverImage: typeof import('./src/components/CoverImage.vue')['default']
+    DesktopLayout: typeof import('./src/components/DesktopLayout.vue')['default']
+    DiscussionBreadcrumbs: typeof import('./src/components/DiscussionBreadcrumbs.vue')['default']
+    DiscussionList: typeof import('./src/components/DiscussionList.vue')['default']
+    DiscussionListByData: typeof import('./src/components/DiscussionListByData.vue')['default']
```

Comment:
A single line change shouldn't affect other lines in a commit. This messes up git blame. If you want to format the other lines, do it in another commit.